tests/contract_testcases/solana/using_functions.sol

```diff
@@ -0,0 +1,31 @@
+contract C {
+	function foo(int256 a) internal pure returns (int256) {
+		return a;
+	}
+}
+
+library L {
+	function bar(int256 a) internal pure returns (int256) {
+		return a;
+	}
+}
+
+library Lib {
+	function baz(int256 a, bool b) internal pure returns (int256) {
+		if (b) {
+			return 1;
+		} else {
+			return a;
+		}
+	}
+	using {L.bar, baz} for int256;
+}
```

Comment:
I think it is good to write a call to these functions so that we check if they are properly wired after sema. Doing something like `uint256 a=1; a.baz(true); a.L.bar;`

Reply:
You are right. Actually I was thinking about this last night and then forgot about it again.